lib/sports_data_api.rb

```diff
@@ -3,6 +3,7 @@
 require "rest_client"
 require "time"
 require "multi_json"
+require "json"
```

Comment:
@RLovelett @mathieujobin Sorry guys, I am completely out of context here.

Did a quick check - If @mathieujobin you are saying that the tests are passing, that should be sufficient. 

By the way, line 38 uses JSON, this PR only adds :: in order to avoid namespace collisions.